packages/database/src/da-beat/current-price/repository.test.ts

```diff
@@ -10,50 +10,39 @@ describeDatabase(CurrentPriceRepository.name, (database) => {
     await repository.deleteAll()
   })
 
-  describe(`${CurrentPriceRepository.prototype.upsert.name} and ${CurrentPriceRepository.prototype.findOneByAssetId.name}`, async () => {
-    it('inserts if not exists', async () => {
-      await repository.upsert(mock('A', 1))
-
-      const currentPrice = await repository.findOneByAssetId('A')
-      expect(currentPrice).toEqual(saved('A', 1))
-    })
-
-    it('updates if exists', async () => {
-      await repository.upsert(mock('A', 1))
-      await repository.upsert(mock('A', 2))
-
-      const currentPrice = await repository.findOneByAssetId('A')
-      expect(currentPrice).toEqual(saved('A', 2))
-    })
-  })
-
-  describe(`${CurrentPriceRepository.prototype.upsertMany.name} and ${CurrentPriceRepository.prototype.findMany.name}`, async () => {
+  describe(CurrentPriceRepository.prototype.upsertMany.name, async () => {
     it('inserts if not exists', async () => {
       await repository.upsertMany([mock('A', 1), mock('B', 2)])
 
-      const currentPrices = await repository.findMany()
+      const currentPrices = await repository.getAll()
       expect(currentPrices).toEqualUnsorted([saved('A', 1), saved('B', 2)])
     })
 
     it('updates if exists', async () => {
       await repository.upsertMany([mock('A', 1), mock('B', 2)])
       await repository.upsertMany([mock('A', 2), mock('B', 3)])
-      const currentPrices = await repository.findMany()
+      const currentPrices = await repository.getAll()
       expect(currentPrices).toEqualUnsorted([saved('A', 2), saved('B', 3)])
     })
   })
 
-  describe(CurrentPriceRepository.prototype.findByIds.name, async () => {
-    it('deletes all currentPrices', async () => {
-      await repository.upsertMany([mock('A', 1), mock('B', 2), mock('C', 3)])
-
-      const currentPrices = await repository.findByIds(['A', 'B'])
-      expect(currentPrices).toEqualUnsorted([saved('A', 1), saved('B', 2)])
-    })
-  })
+  describe(
+    CurrentPriceRepository.prototype.getByCoingeckoIds.name,
+    async () => {
+      it('deletes all currentPrices', async () => {
```

Comment:
nit: let's match the description with the actual test case since we are already here

```suggestion
      it('finds entries by coingeckoIds', async () => {
```